### PR TITLE
feat: add frontend & backend version info in the profile dropdown

### PR DESCRIPTION
**Pull request by Prashanth-Nelli**

### CODE REVIEW

**Comment by moshloop** [comment]
@Prashanth-Nelli the frontend version info should come from package.json

File: src/api/query-hooks/index.ts [hunk 1]
```diff
@@ -23,14 +23,36 @@ import {
...
+        data.data = {
+          ...versionInfo,
+          frontend: packageJson.version,
+          backend: versionInfo.Version
+        };
+        return data;
+      });
+    },
+    {
+      staleTime: defaultStaleTime,
```
Comment (mainawycliffe) [inline src/api/query-hooks/index.ts:50]: Just a nitpick -> I have enabled this globally for all queries https://github.com/flanksource/flanksource-ui/blob/d3f6fb083e72914b46203830f4cc0cabf9ef96ec/src/App.tsx#L52.

File: src/api/query-hooks/index.ts [hunk 2]
```diff
@@ -23,14 +23,36 @@ import {
...
+      enabled: true
```
Comment (mainawycliffe) [inline src/api/query-hooks/index.ts:51]: Another nitpick, shouldn't this be true by default?

File: src/components/VersionInfo/VersionInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { useVersionInfo } from "../../api/query-hooks";
+
+export function VersionInfo() {
+  const { data } = useVersionInfo();
+  const versionInfo = data?.data as any;
```
Comment (mainawycliffe) [inline src/components/VersionInfo/VersionInfo.tsx:5]: We should avoid casting as any and instead provide the necessary types.

**Review by mainawycliffe — APPROVED** [review]
LGTM

**Pull request merged**